boards/adafruit_feather.json

```diff
@@ -0,0 +1,52 @@
+{
+  "build": {
+    "core": "earlephilhower",
+    "cpu": "cortex-m0plus",
+    "extra_flags": "-D ARDUINO_ADAFRUIT_FEATHER_RP2040 -DARDUINO_ARCH_RP2040 -DUSBD_MAX_POWER_MA=250",
+    "f_cpu": "133000000L",
+    "hwids": [
+      [
+        "0x2E8A",
+        "0x00C0"
+      ]
+    ],
+    "mcu": "rp2040",
+    "arduino": {
+      "earlephilhower": {
+        "variant": "adafruit_feather",
+        "boot2_source": "boot2_w25x10cl_4_padded_checksum.S",
+        "usb_vid": "0x239a",
+        "usb_pid": "0x80f1",
```

Comment:
Just curios, how these VID:PID differ from the generic HWIDs above?

Reply:
Hm we the [generator script](https://github.com/earlephilhower/arduino-pico/blob/3bcbb24603aa3ece5a665deeb12a6adfee341931/tools/makeboards.py#L177-L233) always uses the same static HWIDs but then the custom USB VID/PID combination. I guess this is wrong here and the HWIDs field should also always be the `usb_vid` and `usb_pid`? I thought maybe in bootloader mode, it will always revert to this static HWID, but after being programmed, it could change dynamically.

Reply:
OK then, do as you see it fit better.

Reply:
USB VID + PID remained unchanged in `usb_vid` and `usb_pid`, generic HWID unchanged for bootloader compat.